rfcs/feature-set-metadata-file-to-web-platform-tests.md

```diff
@@ -0,0 +1,221 @@
+# RFC #163: Add feature_set metadata file to web-platform-tests
+
+Author: @jcscottiii
+
+# Introduction
+
+The web-platform-tests (WPT) project is a valuable resource for testing web
+platform features and contains valuable metadata such as feature specs.
+Recently, the [WebDX Community Group](https://www.w3.org/community/webdx/)
+began creating the [feature-set](https://github.com/web-platform-dx/feature-set)
+repository. That repository serves as a basic shared catalog of feature
+definitions of the web platform. Feature-set itself doesn't intend to produce
+data but rather link to existing data that will inform audiences which features
+are part of [Baseline](https://web.dev/baseline/). However, there's a great
+opportunity to connect the WPT ecosystem to the feature-set catalog. By doing
+this, it would enable users of wpt.fyi to filter by feature-set grouping, which
+is similar to
+[the ability to filter by spec](https://github.com/web-platform-tests/wpt.fyi/issues/1489).
+The RFC proposes the addition of a FEATURE_SET.yml metadata file which would
+enable the linkage between WPT and feature-set.
+
+# Proposed change
+
+The proposed change includes:
```

Comment:
Presumably there's also a step where this data is integrated in wpt.fyi

Reply:
This will come as a separate RFC. Now that this RFC is moving to separate things (as opposed to #157), there might be a different way to integrate this into wpt.fyi instead of just modifying the current [metadata ingestion path](https://github.com/web-platform-tests/wpt.fyi/blob/9f00b3c20554d7926bbc4ca7877e5b17a59e406e/shared/metadata_util.go#L84).